enhancements/proposals/2549-scylladbcluster-backup-repair/README.md

```diff
@@ -312,7 +457,28 @@ type ScyllaDBManagerTask struct {
 }
 ```
 
-#### Notable changes between `ScyllaDBManagerTask.scylla.scylladb.com/v1alpha1` and legacy `ScyllaClusters.scylla.scylladb.com/v1` and the conversion mechanisms
+#### `ScyllaCluster.scylla.scylladb.com/v1` conversion mechanism
+
+The existing ScyllaCluster (scylla.scylladb.com/v1) controller, running as part of the standalone manager-controller, will be removed. Backup and repair tasks defined in `scyllaclusters.scylla.scylladb.com/v1.spec.backups` and `scyllaclusters.scylla.scylladb.com/v1.spec.repairs` will no longer be reconciled directly.
```

Comment:
duplicate content?

https://github.com/scylladb/scylla-operator/pull/2558/commits/7e8c3eb9e3629ecbbb46540a856031d7e99f667d#diff-6eb3031404c03b7931ca976ae177107512a8cac3e65f6e5a4798aac4a8537a07R266-R268

Reply:
I don't see the duplication - one is about converting the spec into the new API, the other about backpropagating the status.